x/ccv/provider/keeper/proposal.go

```diff
@@ -91,8 +90,8 @@ func (k Keeper) CreateConsumerClient(ctx sdk.Context, chainID string,
 	ts := ctx.BlockTime().Add(k.GetParams(ctx).InitTimeoutPeriod)
 	k.SetInitTimeoutTimestamp(ctx, chainID, uint64(ts.UnixNano()))
 
-	// store LockUnbondingOnTimeout flag
-	if lockUbdOnTimeout {
+	// TODO: this is removed elsewhere
```

Comment:
Is this PR a draft? Or are you not planning to fully remove LockUnbondingOnTimeout in this one?

Reply:
This still needs to be done, right?

Reply:
Added the changes from #551 and closed that PR.